lib/msf/core/db_manager/http/servlet/db_export_servlet.rb

```diff
@@ -0,0 +1,32 @@
+module DbExportServlet
+
+  def self.api_path
+    '/api/v1/db_export'
```

Comment:
Doesn't match Goliath API spec. Resource names should be separated with a dash. Update to be `/api/v1/db-export`.